.github/workflows/ci.yml

```
@@ -26,3 +31,34 @@
+      - name: Install GitHub CLI
+        run: sudo apt-get install -y gh
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Check and Create Pull Request
+        id: create-pr
+        env:
+          GH_TOKEN: ${{ secrets.ACTION_TOKEN }}
+          COMMIT_MESSAGE: ${{ github.event.head_commit.message }}
+        run: |
```

actionlint (1.7.3)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)